### PR TITLE
Add deploy to heroku button

### PR DESCRIPTION
**Pull request by carsongee**
I just verified this works with my devstack.  Super neat.

### CODE REVIEW

**Comment by pwilkins** [comment]
If you are not already logged in to Heroku, the app creation script takes you to your personal apps list following your login.

File: README.rst [hunk 1]
```diff
@@ -27,6 +31,21 @@ project's GitHub site,
 Please see the PyLTI README `https://github.com/mitodl/pylti
 <https://github.com/mitodl/pylti>`_ for a detailed description of the architecture.
 
+Super Quick Start
+-----------------
+
+You can try out the sample app for free by deploying it to your heroku account
```
Comment (pwilkins) [inline README.rst]: nit: consider upper-case: '**h**eroku' -> '**H**eroku'

File: README.rst [hunk 2]
```diff
@@ -27,6 +31,21 @@ project's GitHub site,
...
+simply by clicking the deploy button:
+
+|Deploy|
+
+.. |Deploy| image:: https://www.herokucdn.com/deploy/button.png
+   :target: https://heroku.com/deploy
+
+After deployment is complete you can customize the app from the heroku
+git repository that is created.  This button will also work in forked
+repositories with your own modifications.
```
Comment (pwilkins) [inline README.rst]: Question: Do you want to mention that changing the repository requires changing the app.json file?  Or even pointing to the Heroku page that describes the feature?

Comment (carsongee) [inline README.rst]: Changing the repo should not require changing the app.json file I don't think as it uses the HTTP_REFERRER to determine where to pull the code to deploy from.

**Comment by carsongee** [comment]
How can I require them to login to Heroku first?  Just suggest it in the doc?  I'm guessing this is a bug Heroku will likely fix as it is their ball to fumble, and I would rather not document it as we will likely lose track of the issue.  If you feel it is warranted, I'll add a sentence though.

File: README.rst (hunk 2 above)
Comment (carsongee) [inline README.rst]: I actually just verified this by forking to my account and changing the up template.  Click on the button and building the app lead to the new up message being shown.

File: README.rst (hunk 1 above)
Comment (carsongee) [inline README.rst]: I've upcased it couple places and pushed the change, thanks noticing this.

**Comment by pwilkins** [comment]
Well, the "Deploy..." button simply doesn't do what we want it to do if you have to login to Heroku, though I didn't try to create a new account instead of logging in to my existing account.

**Pull request merged**